api/environments/dynamodb/migrator.py

```diff
@@ -29,11 +30,13 @@ def can_migrate(self) -> bool:
         migration_status = self.migration_status
         return migration_status == ProjectIdentityMigrationStatus.MIGRATION_NOT_STARTED
 
+    def start_migration(self):
```

Comment:
I guess there's a chance now that we'll end up with some locked projects that failed migration but can't be migrated again, right? That's probably ok but might be worth adding a comment?

Reply:
Yep, good shout; I have added the comment.